### PR TITLE
Add firebase tools to gitignore

### PR DESCRIPTION
**Pull request by MvRemmerden**
## Related Issue(s)

Fixes #

## Release Notes

```release-note
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -56,3 +56,5 @@ components/server/*.log
 
 # Jetbrains workspace files
 .idea/misc.xml
+
+.firebase-tools
```
Comment (MvRemmerden) [inline .gitignore:60]:
```suggestion
.firebase-toolssss
```

File: .gitignore [hunk 2]
```diff
@@ -56,3 +56,5 @@ components/server/*.log
```
Comment (mvanremmerden) [inline .gitignore:56]:
```suggestion
Test
```

File: .gitignore [hunk 3]
```diff
@@ -56,3 +56,5 @@ components/server/*.log
 
 # Jetbrains workspace files
 .idea/misc.xml
```
Comment (mvanremmerden) [inline .gitignore:58]:
```suggestion
.idea/misc.xmllll
```

**Review by mvanremmerden — CHANGES_REQUESTED** [review]
Test

File: .gitignore [hunk 4]
```diff
@@ -56,3 +56,5 @@ components/server/*.log
 
 # Jetbrains workspace files
```
Comment (mvanremmerden) [inline .gitignore:57]:
```suggestion
# Jetbrainsy workspace files
```

**Review by mvanremmerden — CHANGES_REQUESTED** [review]

**Review by mvanremmerden — APPROVED** [review]

**Review by mvanremmerden — CHANGES_REQUESTED** [review]
Fix this stupid stuff

File: .gitignore (hunk 2 above)
Comment (mvanremmerden) [inline .gitignore:56]: Test

**Review by mvanremmerden — COMMENTED** [review]
Test

**Pull request merged**